sycl/doc/extensions/experimental/sycl_ext_oneapi_bfloat16_math_functions.asciidoc

```diff
@@ -55,13 +55,16 @@ specification.
 
 == Overview
 
-This extension adds `bfloat16` support to the `fma`, `fmin`, `fmax`, `fabs`
-and `isnan` SYCL floating point math functions. These functions can be used as
-element wise operations on matrices, supplementing the `bfloat16` support
-in the sycl_ext_oneapi_matrix extension.
+This extension adds `bfloat16` support to the `fma`, `fmin`, `fmax`, `fabs`,
```

Comment:
These functions below from SYCL 2020 are missing in the implementation. However, this doesn't block this patch, it can be done in the separate one.

genfloat fmin(genfloat x, sgenfloat y)
genfloat fmax(genfloat x, sgenfloat y)